build-corpus/index.js

```diff
@@ -0,0 +1,33 @@
+import { exec } from 'child_process';
+import { dirname } from 'path';
+import { fileURLToPath } from 'url';
+import { promisify } from 'util';
+
+import { processArgs } from '../lib/process-args.js';
+
+// --- make useful things ---
+
+const execPromise = promisify(exec);
```

Comment:
is this a common pattern for what we're trying to do, using `exec` I mean? I'm not sure I've seen it very frequently.

Reply:
it was a hack.  in the long-run we'd want to export the grabbing and cloning as a module, but it was quicker for me to do it this way today (no need to refactor your code).